frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -279,8 +284,12 @@ function ReaderDictionary:addToMainMenu(menu_items)
                 end,
             },
             {
-                text = _("Download dictionaries"),
-                sub_item_table_func = function() return self:_genDownloadDictionariesMenu() end,
+                text = _("Dictionary presets"),
+                help_text = _("This feature enables you to group (e.g., per language) dictionaries as presets. You can then quickly switch between these presets to change the dictionaries used for lookups. \n\n Note: No other settings are stored in the presets, neither are the actual dictionaries, only a reference to the dictionaries."),
```

Comment:
No spaces around the `\n\n` please. Also an editorial suggestion.
```suggestion
                help_text = _("This feature allows you to organize dictionaries into presets (for example, by language). You can quickly switch between these presets to change which dictionaries are used for lookups.\n\nNote: presets only store a reference to the dictionaries, no other settings."),
```